exec/java-exec/src/test/java/org/apache/drill/exec/store/pcap/TestPcapRecordReader.java

```diff
@@ -45,8 +45,16 @@ public void testStarQuery() throws Exception {
   @Test
   public void testCorruptPCAPQuery() throws Exception {
     runSQLVerifyCount("select * from dfs.`store/pcap/testv1.pcap`", 7000);
-    runSQLVerifyCount("select * from dfs.`store/pcap/testv1.pcap` WHERE is_corrupt=false", 6408);
-    runSQLVerifyCount("select * from dfs.`store/pcap/testv1.pcap` WHERE is_corrupt=true", 592);
+  }
+
+  @Test
+  public void testTrueCorruptPCAPQuery() throws Exception {
+    runSQLVerifyCount("select * from dfs.`store/pcap/testv1.pcap` WHERE is_corrupt=true", 16);
+  }
+
+  @Test
+  public void testNotCorruptPCAPQuery() throws Exception {
+    runSQLVerifyCount("select * from dfs.`store/pcap/testv1.pcap` WHERE is_corrupt=false", 6984);
```

Comment:
Looks like the original test were pretty light. It leaves to the user to test per-column setup and type conversions.

Would recommend adding tests that:

1) Verify the schema: names, types
2) Verifies the data (using the usual mechanism to read a few rows and validate the results.)

Reply:
Concur... I actually found a few undiscovered issues in doing the conversion.  I will add additional unit tests tomorrow.

Reply:
I added new unit tests.  Also, I renamed the `timestamp` column to `packet_timestamp` so that the column name was not a Drill reserved word, which causes issues in queries.